os/include/assert.h

```diff
@@ -70,7 +70,7 @@
 #include <tinyara/compiler.h>
 #include <tinyara/config.h>
 #include <stdint.h>
-
+#define _alert lldbg
```

Comment:
This needs to be removed and _alert needs to be replaced with lldbg or defined in debug.h